net.bartkessels.getit.json

```diff
@@ -0,0 +1,75 @@
+{
+	"app-id": "net.bartkessels.getit",
+	"runtime": "org.gnome.Platform",
+	"runtime-version": "3.24",
+	"sdk": "org.gnome.Sdk",
+	"command": "getit",
+	"finish-args": [
+		"--socket=x11",
+		"--share=ipc",
+		"--socket=wayland",
+		"--share=network",
+		"--filesystem=home",
+		"--filesystem=xdg-run/dconf",
+		"--filesystem=~/.config/dconf:ro",
+		"--talk-name=ca.desrt.dconf",
+		"--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+	],
+	"build-options": {
+		"cflags": "-O2 -g",
+		"cxxflags": "-O2 -g",
+		"env": {
+			"V": "1"
+		}
+	},
+	"cleanup": [
+		"/include",
+		"/lib/pkgconfig",
+		"/man",
+		"/share/doc",
+		"/share/gtk-doc",
+		"/share/man",
+		"/share/pkgconfig",
+		"*.la",
+		"*.a"
+	],
+	"modules": [
+		{
+			"name": "gtksourceview-3.0",
+			"config-opts": ["--disable-Werror"],
+			"builddir": true,
+			"sources": [
+				{
+					"type": "git",
+					"branch": "gnome-3-24",
```

Comment:
this isn't specific enough as a release. it needs a tag or commit as well - or you could just use the archive, it's usually faster.